Library/Homebrew/shims/linux/super/cc

```diff
@@ -192,7 +192,10 @@ class Cmd
       # -R is not the correct way to pass rpaths to the linker
       path = chuzzle($1) || enum.next
       wl = "-Wl," unless mode == :ld
-      args << "#{wl}-rpath=" if keep?(path)
+      if keep?(path)
+        args << "#{wl}--enable-new-dtags"
+        args << "#{wl}-rpath="
+      end
```

Comment:
This change cannot be sent upstream. Since the Android dynamic linker is open source, playing the long game, I'd suggest contributing a patch to Android to add support for `DT_RPATH`. Alternatively, I'd suggest the Termux `ld` tool enable `--enable-new-dtags` by default.